src/helpers.js

```diff
@@ -871,5 +871,6 @@ export function getIfaceSourceName(iface) {
 }
 
 export function canDeleteDiskFile(disk) {
+    console.log(disk);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-650-20230609-135838-a45ed3fa-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)